packages/frontend-api/src/Routing/EnabledOnCurrentDomainChecker.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Shopsys\FrontendApiBundle\Routing;
+
+use Shopsys\FrameworkBundle\Component\Domain\Domain;
+use Shopsys\FrameworkBundle\Component\Domain\Exception\NoDomainSelectedException;
+
+class EnabledOnCurrentDomainChecker
+{
+    /**
+     * @var \Shopsys\FrameworkBundle\Component\Domain\Domain
+     */
+    protected $domain;
+
+    /**
+     * @var int[]
+     */
+    protected $enabledDomainIds;
+
+    /**
+     * @param \Shopsys\FrameworkBundle\Component\Domain\Domain $domain
+     * @param int[] $enabledDomainIds
+     */
+    public function __construct(Domain $domain, array $enabledDomainIds = [])
+    {
+        $this->domain = $domain;
+        $this->enabledDomainIds = $enabledDomainIds;
+    }
+
+    /**
+     * If no domain is selected method returns if it's enabled for the first domain instead.
```

Comment:
I think it would be better to add this info somehow in methods name, because right now this behavior is not clear and you need to read that comment which you might not ...

Reply:
Changed to `isEnabledOnDomain(int $domainId)` in 4d0fda5e0b9ee10fa5af21bba41a3e2763cdf86b